runner.yml.dist

```diff
@@ -30,12 +32,11 @@ drupal:
     - "./vendor/bin/drush en field_ui -y"
     - "./vendor/bin/drush en config_devel -y"
     - "./vendor/bin/drush en toolbar -y"
-    - "./vendor/bin/drush theme:enable oe_theme -y"
-    - "./vendor/bin/drush theme:enable seven -y"
     - "./vendor/bin/drush config-set system.theme default oe_theme -y"
     - "./vendor/bin/drush config-set system.theme admin seven -y"
     - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
     - "./vendor/bin/drush cr"
+    - "./vendor/bin/drush oe-multilingual:import-local-translations"
```

Comment:
Do we need this here, even if we don't test it? We can save a couple of minutes from builds if we don't do this here. We could ship with two runner, one for ci and one for local, if we need this locally, but if we don't test is I doubt we need it at all.